modules/ROOT/pages/mule-exception-strategies.adoc

```diff
@@ -67,4 +73,4 @@ include::{examplesdir}/mule-exception-strategies.xml[]
 == See Also
 
 * *NEXT STEP*: Read on about the structure of a xref:mule-message-structure.adoc[Mule message].
-* Learn more about configuring the xref:catch-exception-strategy.adoc[catch], xref:rollback-exception-strategy.adoc[rollback], xref:reference-exception-strategy.adoc[reference], and xref:choice-exception-strategy.adoc[choice] exception strategies.
\ No newline at end of file
+* Learn more about configuring the xref:catch-exception-strategy.adoc[catch], xref:rollback-exception-strategy.adoc[rollback], xref:reference-exception-strategy.adoc[reference], and xref:choice-exception-strategy.adoc[choice] exception strategies.
```

Comment:
Not addressing any weirdness in the style here via this bug. Just adding new content about the default exception strategy.